cajita/unit_test/CMakeLists.txt

```diff
@@ -110,9 +32,14 @@ set(MPI_TESTS
   SplineEvaluation3d
   SplineEvaluation2d
   Interpolation
+  Parallel
 # BovWriter
   )
 
+if(Kokkos_ENABLE_OPENMPTARGET) #FIXME_OPENMPTARGET
```

Comment:
Is the issue here the hierarchical parallelism?

Reply:
Yes, but we could handle it as discussed #331